imjoy/engine.py

```diff
@@ -18,25 +18,35 @@ def __init__(self, opt, logger):
         self.conn = None
         self.store = None
 
+    def __repr__(self):
+        """Return the engine representation."""
+        return f"<Engine(opt={self.opt})>"
```

Comment:
👍

Reply:
This was handy during testing. It's not completely correct though since we're not including the logger argument. But I plan to remove the logger argument after we change to a logger per module.